src/libpmemobj/palloc.c

```diff
@@ -540,8 +540,9 @@ palloc_exec_actions(struct palloc_heap *heap,
 	 * The operations array is sorted so that proper lock ordering is
 	 * ensured.
 	 */
-	qsort(actv, actvcnt, sizeof(struct pobj_action_internal),
-		palloc_action_compare);
+	if (actv)
```

Comment:
if (!actv || actcnt == 0)
  return;

Reply:
OK, nvm, my fault. Revert it to what you had... And add an assert under that if `UT_ASSERTeq(actcnt, 0);`